server/routes/api/v1/typesRouter.js

```diff
@@ -0,0 +1,16 @@
+import express from "express"
+import Type from '../../../models/Type.js'
+
+const typesRouter = new express.Router()
+
+typesRouter.get("/", async (req, res) => {

```

Comment:
I don't see this endpoint being used? I also don't see a model for Type that would be used in this router